components/tools/OmeroPy/src/omero/__init__.py

```diff
@@ -104,6 +104,14 @@ def proxy_to_instance(proxy_string, default=None):
     """
     import omero
     parts = proxy_string.split(":")
+    # If a default is provided and the format is Class:ID
```

Comment:
@joshmoore I can see that being one possible outcome. Given its low coverage and its current use in the CLI it seems better to use this method to check the valid type. Otherwise the CLI command has to first check the whether the type returned by the proxy is valid. Maybe there is an argument for adding an additional `enforce_default` flag to the method?